run/run.toml

```diff
@@ -28,58 +28,3 @@ node = "../data/lhm/node.arrow"
 edge = "../data/lhm/edge.arrow"
 
 waterbalance = "../data/lhm/waterbalance.arrow"
```

Comment:
Precompile depends on `run`, but that one seems outdated.
The ribasim artifacts don't contain a `forcing-long.arrow` or `waterbalance.arrow`.
https://github.com/visr/ribasim-artifacts/releases/tag/v0.1.0

Reply:
Ah sorry about that. Forcing long is the old long format, so you can just rename it back to forcing.arrow.

Reply:
What should I do with waterbalance? :)

Reply:
Oh that was output, that shouldn't be an issue.

Reply:
Ah I see, thanks :)